### PR TITLE
LPS-67861 Construct navigation using main articles

### PR DESCRIPTION
**Pull request by adolfopa**

### CODE REVIEW

**Comment by adolfopa** [comment]
ci:stop

**Comment by adolfopa** [comment]
ci:retest

File: modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java [hunk 1]
```diff
@@ -1597,10 +1597,10 @@ protected KBArticle getNextKBArticle(
 	}
 
 	protected KBArticle[] getPreviousAndNextKBArticles(KBArticle kbArticle) {
-		List<KBArticle> kbArticles = kbArticlePersistence.findByG_P_S(
+		List<KBArticle> kbArticles = kbArticlePersistence.findByG_P_M(
 			kbArticle.getGroupId(), kbArticle.getParentResourcePrimKey(),
-			WorkflowConstants.STATUS_APPROVED, QueryUtil.ALL_POS,
-			QueryUtil.ALL_POS, new KBArticlePriorityComparator(true));
+			true, QueryUtil.ALL_POS, QueryUtil.ALL_POS,
```
Comment (sergiogonzalez) [inline modules/apps/knowledge-base/knowledge-base-service/src/main/java/com/liferay/knowledge/base/service/impl/KBArticleLocalServiceImpl.java]: We should do the same change in the above method getNextKBArticle and the below method getPreviousKBArticle

**Comment by sergiogonzalez** [comment]
We should also add some integration tests to ensure that this is working as expected.

**Pull request closed without merging**